WireMock/wwwroot/scripts/MatcherUI/HeaderMatcherUI.ts

```
@@ -0,0 +1,56 @@
+            // Add the ignore case cell and its behaviour
+            matcherRow.appendChild(this.renderIgnoreCaseCell(matcher, (e: Event) => {
+                let i = this.getMatcherIndex(matcherRow);
+                let rawMapContent = JSON.parse(rawMap.textContent!);
+                let headerIndex = this.getHeaderIndex(matcherRow);
+                rawMapContent.Request.Headers[headerIndex].Matchers[i].IgnoreCase
+                    = (e.target as HTMLInputElement).checked;
+                rawMap.textContent = JSON.stringify(rawMapContent, null, 1);
+            }));
+
+            // Add the remove button and the behaviour
+            matcherRow.appendChild(this.renderRemoveButtonCell((e: Event) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)